guides/doc-Provisioning_Guide/topics/proc_configuring_iPXE_to_reduce_provisioning_times.adoc

```diff
@@ -0,0 +1,309 @@
+[[Configuring_Networking-Configuring_gPXE_to_Reduce_Provisioning_Times]]
+== Configuring iPXE to Reduce Provisioning Times
+
+Using {Project}, you can configure PXELinux to chainboot iPXE and boot using the HTTP protocol if you have the following restrictions that prevent you from using PXE:
+
+* On networks with unmanaged DHCP servers,
+* If the PXE service is blacklisted on your network or restricted by a firewall
+* When the TFTP UDP-based protocol is not reliable, for example, on low-bandwidth networks
```

Comment:
There is a comma on the first line but not n the others.

Reply:
We will make an English-language editor out of you yet, @lzap  :+1: